truthy-falsy.js

```diff
@@ -0,0 +1,117 @@
+// 1
+const isEmployed = true;
+const isOnVacation = false;
+
+function setAlarm(isEmployed, isOnVacation){
+  return isEmployed && !isOnVacation;
+}
+console.log(setAlarm(isEmployed, isOnVacation));
+
+// 2
+function closeCompare(numberA, numberB, numbersMargin){
+  if (numbersMargin === undefined) numbersMargin = 0;
+  const numbersDiff = Math.abs(numberA - numberB);
+  if (numbersDiff <= numbersMargin) return 0
+  if (numberA < numberB) return -1
+  return 1
+}
+
+// 3
+const or = (a, b) => {
+  if (a == true) return true
+  if (b == true) return true
+  return false
+};
+
+const xor = (a, b) => {
```

Comment:
It would be slightly easier to think about it the other way around.
```
if (a && !b) {
  return true;
}
if (b && !a) {
  return true;
}
return false;
```